### PR TITLE
minor typo

### PR DESCRIPTION
**Pull request by 5HAD0W-P1R4T3**

### CODE REVIEW

**Comment by DanielRuf** [comment]
Not a typo.

Has helped to facilitate ... and empower ... is correct.

**Comment by Trott** [comment]
> Not a typo.
> 
> Has helped to facilitate ... and empower ... is correct.

I think it might be better and more clear as either: 

> Has _helped_ to facilitate ... and _empowered_...

or:

> Has helped _to facilitate_... and _to empower_....

The first option is what this PR does but the second option is probably closer to the intention?

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 [nodejs.org](https://nodejs.org/) by the [OpenJS Foundation](https://openjsf.org/) builds on the merged community's past website projects to form a self-publishing, community-managed version of the previous site.
 
-On a technical level, inspiration has been taken from the `iojs.org` repo while design and content has been migrated from the old [nodejs.org repo](https://github.com/nodejs/nodejs.org-archive). These technical changes have helped to facilitate community involvement and empower the foundation's internationalization communities to provide alternative website content in other languages.
+On a technical level, inspiration has been taken from the `iojs.org` repo while design and content has been migrated from the old [nodejs.org repo](https://github.com/nodejs/nodejs.org-archive). These technical changes have helped to facilitate community involvement and empowered the foundation's internationalization communities to provide alternative website content in other languages.
```
Comment (Trott) [inline README.md:12]: I wonder if the whole paragraph is basically superfluous now and should be removed entirely?

```suggestion
```

**Review by Trott — APPROVED** [review]

**Comment by 5HAD0W-P1R4T3** [comment]
@Trott I think removing the whole paragraph is not a good idea.

**Comment by Trott** [comment]
> @Trott I think removing the whole paragraph is not a good idea.

Why do you think that?

**Comment by Trott** [comment]
@5HAD0W-P1R4T3 Is it OK with you if we reopen this and land it as it is?

**Pull request closed without merging**